### ISSUE TITLE
\left( does not match \left( ... \right) in suggestions. 

### ISSUE THREAD

**Issue by tamuratak** [James-Yu/LaTeX-Workshop#2112]
`\left(` does not match `\left(...\right)` in suggestions. We have to type `\lrparen`.

**Comment by jlelong** [comment]
This is because of #2104 (introduced to fix #2101). 

To put it short, the issue comes from a difference of structure between the `_cmd.json` files and [commands.json](https://github.com/James-Yu/LaTeX-Workshop/blob/cf2eb17b17a9c6617aaed4c9af0b9e8a845a767c/data/commands.json#L23).

In [commands.json](https://github.com/James-Yu/LaTeX-Workshop/blob/cf2eb17b17a9c6617aaed4c9af0b9e8a845a767c/data/commands.json#L23), the dictionary keys may be different from the command itself. For instance,

```
  "lrparen":{
    "command":"left(",
    "snippet":"left(${1}\\right)",
    "detail":"\\left( ... \\right)"
  }
```

whereas in `_cmd.json` file it would be 

```
  "left(":{
    "command":"left(",
    "snippet":"left(${1}\\right)",
    "detail":"\\left( ... \\right)"
  }
```

**Comment by jlelong** [comment]
We still have an issue here as `\left( ... \right)` is proposed as a completion to `\left` but not too `\left(`

![Nov-26-2020 15-34-24](https://user-images.githubusercontent.com/2910140/100363362-e7135500-2ffc-11eb-81d4-f0bb26667e77.gif)
